buildSrc/src/main/kotlin/Extensions.kt

```diff
@@ -24,6 +24,21 @@ internal val PluginContainer.kotlinKapt: Unit
         apply("kotlin-kapt")
     }
 
+internal val PluginContainer.kotlinLibrary: Unit
+    get() {
+        apply("kotlin-library")
+    }
+
+internal val PluginContainer.javaLibrary: Unit
+    get() {
+        apply("java-library")
+    }
+
```

Comment:
this is not the name of the kotlin plugin tho. You need to recheck your implementation as this wouldn't work